boards/wio_terminal/Cargo.toml

```diff
@@ -31,33 +31,38 @@ chip = "ATSAMD51P19A"
 
 [dependencies]
 bitfield = "0.13"
-cortex-m = "0.7.3"
 cortex-m-rt = { version = "0.7", optional = true }
-display-interface-spi = "0.4"
+
+display-interface-spi = "0.5"
 heapless = "0.8"
-ili9341 = "0.5.0"
+ili9341 = "0.6.0"
 lis3dh = "0.3.0"
 embedded-sdmmc = "0.3.0"
-usb-device = { version = "0.2", optional = true }
+usb-device = { version = "0.3", optional = true }
+nb = "1.0"
 bbqueue = { version = "0.5", optional = true }
 generic-array = { version = "0.14", optional = true }
 seeed-erpc = { version = "0.1.2", optional = true }
 
+[dependencies.cortex-m]
+features = ["critical-section-single-core"]
+version = "0.7"
+
 [dependencies.atsamd-hal]
-version = "0.16.0"
+path = "../../hal"
```

Comment:
Please make sure to revert this dependency to the `crates.io` dependency, as `wio_terminal` is a Tier 2 BSP.

Reply:
I will, this was just giving me an error due to missing PanicOnRead. Thanks